201-blob-storage-private-endpoint/scripts/nslookup.sh

```diff
@@ -1,21 +1,35 @@
 #!/bin/bash
 
 # Variables
-blobServicePrimaryEndpoint=$1
+adlsServicePrimaryEndpoint=$1
+blobServicePrimaryEndpoint=$2
+
+# Parameters validation
+if [[ -z $adlsServicePrimaryEndpoint ]]; then
+    echo "adlsServicePrimaryEndpoint parameter cannot be null or empty"
+    exit 1
+fi
 
-# Parameter validation
 if [[ -z $blobServicePrimaryEndpoint ]]; then
     echo "blobServicePrimaryEndpoint parameter cannot be null or empty"
     exit 1
 fi
 
+# Eliminate debconf: warnings
+echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
 # Update the system
 sudo apt-get update -y
 
 # Upgrade packages
 sudo apt-get upgrade -y
 
+# Run nslookup to verify that the <storage-account>.dfs.core.windows.net public hostname of the storage account 
+# is properly mapped to <storage-account>.privatelink.dfs.core.windows.net by the privatelink.dfs.core.windows.net 
```

Comment:
Can we make the comments AzureGov friendly?  Here and below?

Reply:
As above, done :)